### PR TITLE
CY-412: Moving to CircleCI V2.0

### PR DESCRIPTION
**Pull request by tyacbovi**
Delete obsolete files

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      - checkout
+      - run:
+          name: Install tox, NOTICE we use an old version of tox because of CFY-6398 ( relying dict ordering)
+          command: sudo pip install tox==1.6.1
+      - run:
+          name: Run tox of specfic environment
+          command: tox -e $DO_ENV
+
+  - &test_defaults_for_python26
+    docker:
```
Comment (opencm) [inline .circleci/config.yml]: Should be -
image: circleci/python:2.6
right?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <<: *test_defaults
+    environment:
+      DO_ENV: test_py27
+
+  test_py26:
+    <<: *test_defaults_for_python26
+    environment:
+      DO_ENV: test_py26
+
+  docs:
```
Comment (opencm) [inline .circleci/config.yml]: no need to create docs - it is generating under cloudify-common

File: .circleci/config.yml (hunk 1 above)
Comment (tehasdf) [reply .circleci/config.yml]: circle doesn't provide a 2.6 image anymore but we do `pyenv install 2.6.9` and `pyenv local 2.6.9` to install 2.6

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+workflows:
+  version: 2
+
+  build_and_test:
+    jobs:
+      - flake8
+      - test_py27
+      - test_py26
+      - docs
```
Comment (opencm) [inline .circleci/config.yml]: remove the docs

**Pull request merged**